Лаб_работа 1/task_1.py

```diff
@@ -0,0 +1,5 @@
+numbers = [2, -93, -2, 8, None, -44, -1, -85, -14, 90, -22, -90, -100, -8, 38, -92, -45, 67, 53, 25]
+
+# TODO заменить значение пропущенного элемента средним арифметическим
+numbers[4] = (sum(numbers[:4:]) + sum(numbers[5::])) / len(numbers)
```

Comment:
```suggestion
numbers[4] = (sum(numbers[:4]) + sum(numbers[5:])) / len(numbers)
```
Лишнее двоеточие. При слайсировании достаточно указания одного индекса границы